redhat-developer/service-binding-operator
# Introduce SBO logo

### Motivation

Currently SBO uses Red Hat official logo as it's own.

### Changes

This PR replaces the Red Hat logo with suggested SBO logo:
| Normal | Small |
|---|---|
|![image](https://user-images.githubusercontent.com/3263627/114050163-52631280-988c-11eb-8d2f-8ea8ef3fe645.png)|![image](https://user-images.githubusercontent.com/3263627/114050280-6c045a00-988c-11eb-97f9-dbbc45ee4b8f.png)|


### Testing
* Build and push operator's bundle and index image by running `release-operator`
* Create a CatalogSource referencing the above index image and install it into the OpenShift's marketplace
* The OpenShift's OperatorHub should list the the custom SBO with the new logo like the following:
![image](https://user-images.githubusercontent.com/3263627/114050093-437c6000-988c-11eb-9f50-bc4efa1c21a8.png)
